android/app/src/main/java/com/reactnativenavigation/views/RnnToolBar.java

```diff
@@ -56,6 +56,7 @@
     private DrawerLayout mDrawerLayout;
     private ActionBarDrawerToggle mDrawerToggle;
     private ArrayList<View> mMenuItems;
+    private int mButtonColor;
```

Comment:
@jskuby Lets not keep this as a member since we use it only in a single function